src/unity/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -221,7 +226,6 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
             (8.0, 16.0), (8.0, 8.0), (16.0, 8.0),
             (16.0, 32.0), (16.0, 16.0), (32.0, 16.0),
         ],
-        'grid_shape': [13, 13],
```

Comment:
In case this parameter is needed elsewhere outside this function, it's probably safer to keep it in the `params` dictionary (which gets passed around to various other places). You could keep it as  `'grid_shape': grid_shape` to use the user-provided value.